.fixtures.yml

```diff
@@ -1,19 +1,19 @@
 ---
 fixtures:
   repositories:
-    auditd: https://github.com/simp/pupmod-simp-auditd.git
+    auditd:    'https://github.com/simp/pupmod-simp-auditd.git'
```

Comment:
Many of our modules don't use quotes around the strings in .fixtures.yml.  Why here?  Do we have a coding standard that requires them?  If not, I wouldn't make these changes.

Reply:
@lnemsick-simp It was a copy/pasta from another module. It's pretty much arbitrary at this point but I'll switch it back.